ivadomed/utils.py

```diff
@@ -946,3 +946,10 @@ def __call__(self, sample):
                 return False
 
         return True
+
+
+def unstack_tensors(sample):
+    list_tensor = []
+    for i in range(sample.shape[1]):
+        list_tensor.append(sample[:, i, ].unsqueeze(1))
+    return list_tensor
```

Comment:
To move to `transforms.py`?

Reply:
I put unstack_tensors as transform at first, but because of the collate (to create batches) I wasn't able to get the right format. I got a list of length batch_size, but what I actually want is a list of length of the number of modality with the shape: batch_size, 1, H, W. Since it is not a transform, I would not put it in transforms, but there might be a better place to put it than utils.py